master/struct.html

```diff
@@ -905,49 +1028,303 @@ <h3 id="UseAttributes">Attributes</h3>
+The user agent must once again attempt to resolve the URI to identify the referenced element.
+</p>
+  
+<p class="issue">
+  Should there be some sort of load event
+  that is dispatched after the use tree is successfully generated?
+  If so, which event in particular?
+</p>
 
+<h3 id="UseShadowTree">The use-element shadow tree</h3>
+  
+<p>The <a>re-used graphics</a> generated by a <a>'use'</a> element
+are defined in terms of a <a>shadow tree</a>.
+In terms of interactivity and style inheritance,
+they are therefore quite different from other types of <a>re-used graphics</a> in SVG,
+such as <a>'pattern'</a> and <a>'marker element'</a> content.
+</p>
+  
+<p>
+Elements in the shadow tree are rendered as if 
+the <a>'use'</a> element was a container and they were its children.
+However, the SVG Document Object Model (DOM) only contains 
+the <a>'use'</a> element and its attributes. 
+The SVG DOM does not include the
+element instances as children of the <a>'use'</a> element.
+</p>
+  
+<p>User agents that support scripting and the document object model
+must implement the <a>use-element shadow tree</a> as described in this section.
+In contrast, user agents that do <em>not</em> support 
+the dynamic interactive processing mode 
+may not need to implement all the details of the shadow DOM. 
+However, all user agents must ensure that the <a href="#UseLayout">layout</a> 
+and <a href="#UseStyleInheritance">style inheritance</a> for the re-used graphics,
+and any <a href="#UseMultimedia">multimedia</a> 
+and <a href="#UseAnimations">declarative animations</a> if applicable,
+are rendered in the same way as if the shadow DOM was implemented. 
+</p>
+  
+<p>The following definitions apply when discussing <a>'use'</a> elements 
+  and their shadow trees:</p>
+
+<dl>
+  <dt id="TermReferencedElement">referenced element</dt>
+  <dd>The element specified by the <a>'href'</a> (or <a>'xlink:href'</a>) attribute on the <a>'use'</a> element, or the root element of a document referenced by that attribute if the URL provided does not include a target fragment that links to a specific element <a>'id'</a>.</dd>
+  
+  <dt id="TermReferencedDocumentSubtree">referenced document subtree</dt>
+  <dt>referenced graphics</dt>
+  <dd>The referenced element, and all of its descendent nodes.</dd>
+  
+  <dt id="TermShadowRoot">shadow root</dt>
+  <dd>A type of <a>DocumentFragment</a> node which is associated with a host <a>Element</a>,
+  and which contains the content that will be used to render that host.
+  A shadow root should be implemented in conformance with the <a href="https://www.w3.org/TR/shadow-dom/">Shadow DOM</a> specification <a href="refs.html#ref-SHADOWDOM">[SHADOWDOM]</a>,
+  or its future replacement.
+  </dd>
+  
+  <dt id="TermShadowHost">shadow host</dt>
+  <dt>host</dt>
+  <dd>An element that has an associated shadow root.</dd>
+  
+  <dt id="TermShadowTree">shadow tree</dt>
+  <dd>A node tree whose root is a shadow root.</dd>
```

Comment:
Can we just link to the DOM definition of a shadow tree?
https://dom.spec.whatwg.org/#shadow-trees